src/config.cpp

```diff
@@ -150,6 +150,14 @@ Config& Config::parse(int argc, char** argv) {
                 "Flag indicating model is stateful",
                 cxxopts::value<bool>()->default_value("false"),
                 "STATEFUL")
+            ("metrics_enabled",
+                "Flag indicating if metrics are enabled",
+                cxxopts::value<bool>()->default_value("false"),
+                "METRICS")
```

Comment:
We don't validate that if someone requests metrics enabling and does not pass rest port then it does not make any sense.

Reply:
Good point lets not allow this at start.